app/src/main/java/com/github/sdp/mediato/model/User.java

```diff
@@ -0,0 +1,127 @@
+package com.github.sdp.mediato.model;
+
+import com.github.sdp.mediato.errorCheck.Preconditions;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class User {
+    public static final int LIMIT_LOCATION_SIZE = 2;
+    public static final String FAVOURITES_COLLECTION = "Favourites";
+    public static final String RECENTLY_WATCHED_COLLECTION = "Recently Watched";
+
+    private String id = "";
+    private String username = "";
+    private String displayedName = "";
+    private String email = "";
+    private String birthDate = "";
+    private String registerDate = "";
+
+    private List<Double> location = new ArrayList<>();

```

Comment:
I initially wanted to use an array with fixed size but those are not accepted by Firebase. We could create a Location class but that sounds a bit overkill.

Reply:
I think a Location class would be a useful abstraction to define such methods as `getX()` because from a list of double it is not obvious how to extract the coordinates for example.

Reply:
Good point, adressed in commit [22e38d9].